docker/README.md

```diff
@@ -5,46 +5,88 @@ Thats right folks onedrive is now dockerized ;)
-docker inspect onedrive > /dev/null && docker rm -f onedrive
-docker run $firstRun --restart unless-stopped --name onedrive -v onedrive_conf:/onedrive/conf -v "${onedriveDir}:/onedrive/data" driveone/onedrive
+This will create a docker volume labeled 'onedrive_conf', which we will use it later.
+
+The second one is your data folder that needs to sync with. The owner of the folder must not be root, and you must have permission to its parent directory (because onedrive will try to setup a monitor for the sync folder).
+
+### 3. First run
+
+Onedrive also needs to be authorized with your account. This is done by running docker in interactive mode. 
+
+**make sure to change onedriveDir to your own.**
+```bash
+onedriveDir="${HOME}/OneDrive"
+docker run -it --restart unless-stopped --name onedrive -v onedrive_conf:/onedrive/conf -v "${onedriveDir}:/onedrive/data" driveone/onedrive
 ```
-## Poweruser section
-1.  Check if monitor service is running
+
+You will be asked to open a specific link using your web browser where you will have to login into your Microsoft Account and give the application the permission to access your files. After giving the permission, you will be redirected to a blank page. Copy the URI of the blank page into the application.
+
+If your onedrive is working as expected, you can detach from the container with Ctrl+p, Ctrl+q.
+
+### 4. Status, stop, and restart
+
+Check if monitor service is running
 ```bash
 docker ps -f name=onedrive
 ```
-2.  Show monitor run logs
+
+Show monitor run logs
 ```bash
 docker logs onedrive
 ```
-3.  Stop running monitor
+
+Stop running monitor
 ```bash
 docker stop onedrive
 ```
-4.  Resume monitor
+
+Resume monitor
 ```bash
 docker start onedrive
 ```
-5.  Unregister onedrive monitor
+
+Unregister onedrive monitor
 ```bash
 docker rm -f onedrive
 ```
+
+### 5. Edit the config
+
+Onedrive should run in default configuration, but however you can change your configuration. First download the default config from [here](https://raw.githubusercontent.com/abraunegg/onedrive/master/config) and put it into your onedrive_conf volumn. The detailed document can be found here: [additional-configuration](https://github.com/abraunegg/onedrive#additional-configuration)
+
+## Run or update with one script
+If you are experienced with docker and onedrive, you can use the following script:
+```bash
+# Update onedriveDir with correct existing OneDrive directory path
+onedriveDir="${HOME}/OneDrive"
+
+firstRun='-d'
+docker pull driveone/onedrive
+docker inspect onedrive_conf > /dev/null || { docker volume create onedrive_conf; firstRun='-it'; }
+docker inspect onedrive > /dev/null && docker rm -f onedrive
+docker run $firstRun --restart unless-stopped --name onedrive -v onedrive_conf:/onedrive/conf -v "${onedriveDir}:/onedrive/data" driveone/onedrive
+```
+
 ## Build instructions
 ```bash
-cd docker
 git clone https://github.com/abraunegg/onedrive
+cd onedrive/docker
 docker build . -t driveone/onedrive
```

Comment:
This change is incorrect. We need the `onedrive` directory below the `docker` dir. If you want to give instructions starting from 0, then the correct would be
```
git clone https://github.com/abraunegg/onedrive
cd onedrive/docker
git clone https://github.com/abraunegg/onedrive
docker build . -t driveone/onedrive
```

Mind the double clone - could be circumvented with cp -a or something similar, too.

Reply:
That I am all for it, moving the Dockerfile to the root and incorporating the docker descriptions into the main readme!!!